src/App.js

```diff
@@ -61,40 +63,30 @@ function App() {
 
     return (
         <>
-            
             <BrowserRouter>
+            <Header
+                toggleDarkMode={toggleDarkMode}
+                isDarkModeTheme={isDarkModeTheme}
+                onShowModal={handleDisplayingModal}
+            />
+            <WelcomeBar/>
```

Comment:
Would see this in the `Layout` component.

Layout component has this skelatone: 

```
const Layout = () => {
return (
<div className="layout">
    <Header />

    <main className="content">
      {Outlet}
    </main>

   <Footer />
</div>
)

}
```